lib/active_model/serializer.rb

```diff
@@ -177,6 +177,10 @@ def json_key
       end
     end
 
+    def as_json
+      self.class.adapter.new(self).as_json
```

Comment:
@ryansch  ```.create``` calls ```.new``` behind the scenes
The difference here would be if you want to set the adapter through options.

IMO it's safe to assume that the adapter can be presumed.
btw if it was ```.create``` it would need to pass the resource instead of self
If there is a specific reason that I'm haven't seen, just let me know! :smile: